ody_flutter/lib/screens/gatherings/gatherings_screen.dart

```diff
@@ -41,32 +41,59 @@ class _GatheringsScreenState extends State<GatheringsScreen> {
         backgroundColor: CommonColors.cream,
         floatingActionButton: _buildFloatingActionButton(),
         body: SafeArea(
-          child: Column(
+          child: Stack(
             children: [
+              if (_viewModel.gatherings.isEmpty)
+                _buildEmptyGathering()
+              else
+                Padding(
```

Comment:
요거 Stack 안쓰고 이런 식으로는 안되나요 ?
```suggestion
          child: (_viewModel.gatherings.isEmpty) 
              ? _buildEmptyGathering() 
              : Padding(~
```

Reply:
이게 오디 탑 바가 맨 위에 있어야 해가지구..